### ISSUE TITLE
Freezing & Voting on Tronscan

### ISSUE THREAD

**Issue by UnBuenCamino** [tronscan/tronscan-frontend#1143]
Freezing and voting on Windows 10 machines using Tronscan.org has not worked properly for AT LEAST OVER ONE YEAR!!! Many people have documented this flaw on Github.  Why is no one interested in addressing it and fixing it.  For weeks, I have been trying to freeze new Tron and trying to vote already frozen Tron.  It used to work intermittently, now, it just doesn't work 100% of the time.

**Comment by Rovak** [comment]
Voting and freezing works fine on tronscan.org, i've have done it several times in the past week. If you are using the tronscan desktop client then please use tronscan.org instead

**Comment by UnBuenCamino** [comment]
This is the typical superficial response Tron always provides. Essentially, you have NO MEANINGFUL SUPPORT.

As I stated in all of my various emails and GitHub posts, I do use Tronscan.org. I run it on a Windows 10 Professional Dell computer using a Ledger Nano S login. Freezing and voting do for work for me and, based in GitHub posts, do not work for other people as well.

Tron's position of "it works fine for me therefore it should work fine for you" or "since it works fine for me, we will not help you solve your problem" is very frustrating. It does not constitute technical support by any standard.

To repeat, I have no doubt that freezing and voting works for you. What I or the other GitHub posters can't communicate is that even though it works for you, it does not work for us. I have a substantial amount of Tron. Do you have any higher level technical support to help substantial Tron holders resolve serious technical issues?

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by Rovak** [comment]
Issues with ledger usually comes down to the following causes:

- A chrome update broke something, try a different opera like Opera
- Your ledger isn't updated. Try updating your ledger and updating the TRON app
- An extension is causing problems

There is a telegram channel where you can have more direct support: https://t.me/tronscan_org

**Comment by UnBuenCamino** [comment]
I've tried Chrome, Brave and Firefox. I've updated to the latest bios for Ledger, I've uninstalled and reinstalled the Tron app in Ledger (after bios update). I'll look into t.me.tronscan.org. However, prior to the Ledger updates, although Ledger login always worked, freezing and voting hardly ever worked. For both freezing and voting and Ledger login, I've seen lots of GitHub posts and lots of Reddit discussion of both issues. It's clear to me that both Tron and Ledger have significant quality control issues and the ability of these two large entities to coordinate technical support in any meaningful way is lacking. I would like to see Tron accelerate implementing support for Trezor. I would also like to see Tron/Ledger and Tron/Trezor implement direct native support like is done for Bitcoin Core, Bitcoin Cash and XRP. This approach is vastly superior to the way you are now supporting Ledger. These are the kinds of things that can really separate Tron from the rest of the field, especially from EOS.

Tron has a lot of work to do in this area. One small thing that would help immensely is better error handling. When something doesn't work, like freezing, voting, or login, error handling is frequently simply 100% absent and when it does exist, it is not helpful. Meantime, thanks for your suggestion, I'll check it out.

Sent from ProtonMail mobile

-------- Original Message --------
